content/actions/creating-actions/metadata-syntax-for-github-actions.md

```diff
@@ -259,6 +259,36 @@ For more information, see "[`github context`](/actions/reference/context-and-exp
 **Required** The shell where you want to run the command. You can use any of the shells listed [here](/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell). Required if `run` is set.
 {% endif %}
 
+#### `runs.steps[*].if`
+
+**Optional** You can use the `if` conditional to prevent a step from running unless a condition is met. You can use any supported context and expression to create a conditional.
```

Comment:
`You can use any supported context and expression to create a conditional.` This is easy for me to understand because i'm really close to the code, but for users who aren't that familiar with actions they may not know that we specifically do not allow you to access the `secrets` in composite actions, would it be good to call our or mention that here again just for clarity?

Reply:
Do we say somewhere else that `secrets.*` is not supported in composite actions? I feel the [main](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) composite page would be a better place to make that clear.

Reply:
Maybe here? https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability

Doesn't have to be done in this pr

Reply:
> 

It would be super helpful to document what context are not supported in [the contexts article](https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability) (and then to link to it here and from the [creating a composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) article).

Since you are a Hubber, you can [open an issue in docs-content](https://github.com/github/docs-content/issues/new/choose) and the docs team can work with you to document this info. 

(cc @martin389 for the Actions docs improvement project)